docs/ancestry.md

```diff
@@ -730,14 +739,13 @@ ts = msprime.sim_ancestry(
 ts
 ```
 
-To isolate tree sequences for each chromosome, we can trim the full tree sequence
-for each chromosome by defining the end points of each chromosome and using
-the {meth}`~tskit.TreeSequence.keep_intervals` method.
-It is important to specify `simplify=False` so
-that node indices remain consistent across chromosomes. The
-{meth}`~tskit.TreeSequence.trim` method
-is then used to trim the tree sequences to the focal chromosome, so that position
-indices begin at 0 within each chromosome tree sequence.
+To isolate tree sequences for each chromosome, we can trim the full tree
+sequence for each chromosome by defining the end points of each chromosome and
+using the {meth}`~tskit.TreeSequence.keep_intervals` method. It is important to
+specify `simplify=False` so that node indices remain consistent across
+chromosomes. The {meth}`~tskit.TreeSequence.trim` method is then used to trim
+the tree sequences to the focal chromosome, so that position indices begin at
+0 within each chromosome tree sequence.
 
 ```{code-cell}
 chrom_positions = [0, 1e6, 2e6, 3e6]
```

Comment:
I know this isn't code you touched, but... perhaps `chrom_positions` should be moved above and used to define `positions`.

Reply:
This was actually an example I put together before, so happy to clean it up so it makes more sense! I moved a few things around so hopefully it helps a bit.